README.md

```diff
@@ -33,7 +33,7 @@ Support for running FireCloud outside of the Broad is planned.
 
 ### Development Stack Setup
 
-1. Make sure you've run the `firecloud-setup.sh` script in the `firecloud-develop` repo to render the config files and run script.
+1. Make sure you've run the `firecloud-setup.sh` script in the `local` branch of the `firecloud-develop` repo to render the config files and run script.
```

Comment:
Checking with @zarsky-broad to see what to update this to. Last I checked with him, there's a new script in the develop branch (maybe all branches) called `render-configs.sh` that also calls the firecloud-setup script.